lib/block-patterns.php

```diff
@@ -70,3 +70,86 @@
 						<!-- /wp:query -->',
 	)
 );
+
+register_block_pattern(
+	'query/grid-posts',
+	array(
+		'title'      => __( 'Grid', 'gutenberg' ),
+		'blockTypes' => array( 'core/query' ),
+		'categories' => array( 'Query' ),
+		'content'    => '<!-- wp:query {"query":{"perPage":6,"pages":0,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"exclude","inherit":true},"layout":{"type":"flex","columns":3}} -->
```

Comment:
`perPage:6` seems a bit big too me for preview, that of course it could be changed anytime... Do you think it could 'work' with three?

Reply:
I tried three, but it doesn't show a grid of posts then, just a row of three. Since it's called "Grid", I think it makes sense to show both a row and a column.